.pre-commit-config.yaml

```diff
@@ -19,6 +19,6 @@ repos:
   - repo: git://github.com/antonbabenko/pre-commit-terraform
     rev: v1.12.0
     hooks:
-      - id: terraform_docs
+      # - id: terraform_docs
       - id: terraform_fmt
       - id: terraform_validate_no_variables
```

Comment:
I think you need to run `pre-commit autoupdate` on this repo and also remove `terraform_validate_no_variables`.